protocol/flows/src/lib.rs

```diff
@@ -4,3 +4,4 @@ pub mod flowcontext;
 pub mod service;
 pub mod v5;
 pub mod v6;
+pub mod v7;
```

Comment:
I would do the following and delete v7/mod.rs

```suggestion
pub use v6 as v7;
```